geomesa-tools/src/main/assemblies/component.xml

```diff
@@ -15,8 +15,24 @@
                 <exclude>org.locationtech.geomesa:geomesa-accumulo-distributed-runtime*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-compute*</exclude>
                 <exclude>com.google.guava:guava</exclude> <!-- Guava conflicts with Hadoop 2.x which provides guava 11.0.2-->
+                <exclude>org.apache.zookeeper:zookeeper</exclude>
+                <exclude>org.apache.kafka:kafka_2.11</exclude>
             </excludes>
         </dependencySet>
+        <dependencySet>
+            <outputDirectory>lib/kafka</outputDirectory>
+            <directoryMode>755</directoryMode>
+            <fileMode>0644</fileMode>
+            <useProjectArtifact>false</useProjectArtifact>
+            <useTransitiveDependencies>true</useTransitiveDependencies>
+            <useTransitiveFiltering>true</useTransitiveFiltering>
+            <includes>
```

Comment:
picky...try to exclude things in ```lib/kafka``` that are already in ```lib/common```:
```
ahulbert@doubletrouble:/tmp/geomesa-tools-1.2.2-SNAPSHOT$ ls lib/kafka/ -1
commons-lang3-3.3.2.jar
geomesa-feature-kryo-1.2.2-SNAPSHOT.jar
geomesa-kafka-datastore-1.2.2-SNAPSHOT.jar
guava-17.0.jar
jopt-simple-3.2.jar
kafka_2.11-0.8.2.1.jar
kafka-clients-0.8.2.1.jar
lz4-1.2.0.jar
metrics-core-2.2.0.jar
zkclient-0.3.jar
zookeeper-3.4.5.jar
```
I think it would be excluding:
```
lib/common/commons-lang3-3.3.2.jar
lib/common/geomesa-feature-kryo-1.2.2-SNAPSHOT.jar
```
i excluded the kafka ds of course